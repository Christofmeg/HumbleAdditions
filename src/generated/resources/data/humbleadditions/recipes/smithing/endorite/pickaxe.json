{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorite_block"
  },
  "base": {
    "item": "humbleadditions:endorium_pickaxe"
  },
  "result": {
    "item": "humbleadditions:endorite_pickaxe"
  }
}