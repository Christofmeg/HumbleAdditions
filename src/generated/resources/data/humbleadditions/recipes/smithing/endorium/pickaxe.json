{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorium_block"
  },
  "base": {
    "item": "minecraft:diamond_pickaxe"
  },
  "result": {
    "item": "humbleadditions:endorium_pickaxe"
  }
}