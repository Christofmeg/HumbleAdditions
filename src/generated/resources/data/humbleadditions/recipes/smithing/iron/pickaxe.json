{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_ingot"
  },
  "base": {
    "item": "minecraft:golden_pickaxe"
  },
  "result": {
    "item": "minecraft:iron_pickaxe"
  }
}