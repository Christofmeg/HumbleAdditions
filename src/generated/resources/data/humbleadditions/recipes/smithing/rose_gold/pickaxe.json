{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:rose_gold_ingot"
  },
  "base": {
    "item": "minecraft:golden_pickaxe"
  },
  "result": {
    "item": "humbleadditions:rose_gold_pickaxe"
  }
}