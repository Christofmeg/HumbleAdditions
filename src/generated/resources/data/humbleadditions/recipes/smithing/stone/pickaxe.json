{
  "type": "minecraft:smithing",
  "addition": {
    "tag": "minecraft:stone_tool_materials"
  },
  "base": {
    "item": "minecraft:wooden_pickaxe"
  },
  "result": {
    "item": "minecraft:stone_pickaxe"
  }
}