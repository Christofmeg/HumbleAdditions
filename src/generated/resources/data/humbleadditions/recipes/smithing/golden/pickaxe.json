{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_ingot"
  },
  "base": {
    "item": "minecraft:stone_pickaxe"
  },
  "result": {
    "item": "minecraft:golden_pickaxe"
  }
}